.clang-tidy

```diff
@@ -61,6 +61,5 @@ readability-simplify-subscript-expr,
 readability-string-compare,
 '
 HeaderFilterRegex: '^(aten/|c10/|torch/).*$'
-AnalyzeTemporaryDtors: false
```

Comment:
While this option does nothing, I think it's still dumped out by clang-tidy. Is it being removed in a future version and causing a hard error or something?

Reply:
Yes, clang-tidy 18 reports an error.